### PR TITLE
New transaction logic

### PR DESCRIPTION
**Pull request by xjakubo**
I changed transaction logic, changed string formatting style in these (using fstrings)

-implemented roundDown function to use all of available funds, simply cutting decimal places to stepSize of coin (ignored small coins).
-moved order to function for better visibility.
-added order_quantity to try function, so if coin blows up it adjusts price.
-added try-catch to all network functions.

### CODE REVIEW

File: crypto_trading.py [hunk 1]
```diff
@@ -77,8 +77,24 @@ def format(self, record):
...
+'TOMO': 100,
+'EOS': 100,
+'QTUM': 1000,
+'ETC': 100,
+'XMR': 100000,
+'DASH': 100000,
+'NEO': 1000,
+'ATOM': 1000,
+'OMG': 100
+}
```
Comment (german-bortoli) [inline crypto_trading.py:97]: Why is this needed? Can't be something more dynamic than hardcoding it ?

**Comment by edeng23** [comment]
@xjakubo Are you still sure that this is necessary and not part of the code already?

**Comment by rel-s** [comment]
This PR will be closed for now until it is explaind & rebased.
